Support.md

```diff
@@ -81,3 +81,30 @@ The Senior Support Engineer role builds upon the foundation of the Support Engin
 - I am actively introspective and take time to work on my personal and professional development.
 - I happily take responsibility for my own mistakes and I learn from the mistakes of others.
 
+## Lead Support Engineer
+
+The Lead Support Engineer role builds upon the foundation of the Support Engineer and Senior Support Engineer roles.
+
+> I am a technical leader, a person that is recognised both within the Support team and as part of the wider Octopus community for the breadth and depth of your technical ability.
+
+### I am trusted
+
+- I have mastered my tools and the core building blocks used by our team.
+- I have a a deep technical understanding of more than one area of Octopus, and can confidently use this knowledge to assist customers and fellow support team members in all areas of Octopus.
+- I know when it makes sense to push back against a customer, and when it is appropriate to not create an issue
+- I know how to mentor a customer to the best path, and understand that what they are asking for may not be what they need
+
```

Comment:
Thinking about the progression from Support Engineer to Senior Support Engineer and on to Lead Support Engineer, I think it might be good to include something like:
```suggestion
- I am trusted across Octopus Deploy and have established good working relationships across the organization, and I co-ordinate our response to complicated support issues by pulling in the right people at the right time.
```